src/bundle/test-transformer.ts

```diff
@@ -35,7 +36,6 @@ export const setupTransformHook = (): VoidFunction => {
                     },
                 }),
             ],
-            require("@babel/plugin-transform-typescript"),
```

Comment:
In some cases `@babel/plugin-transform-typescript` works incorrectly with my tsx files. But if I use @babel/preset-typescript then everything works fine. I know that this preset just use this plugin but it looks like preset run after plugins due to the docs. So I just rewrite code to use preset instead this plugin.